mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,15 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
     return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+return "Hello, my name is " + name + " and I am " + age + " years old";
+}
 
 function getTotal(a, b) {
-    total = a ++ b;
-
-    return "The total is total"
+   return total = "The total is " + (a + b);
```

Comment:
The test cases passed and syntatically correct. However, the code is confusing. You are either returning a variable or a statement. But in this line, you are returning a variable AND assigning this variable to a string. Can you please modify this code so that it only does one thing: return a variable OR return a string/statement, but not both. You can solve this in max 2 lines

Reply:
Sorry, I don't understand. Thank you

Reply:
We can discuss this in the meeting tomorrow